src/components/CommentCard.jsx

```diff
@@ -1,14 +1,29 @@
-import Card from "@mui/material/Card";
 import CardActions from "@mui/material/CardActions";
 import CardContent from "@mui/material/CardContent";
 import Typography from "@mui/material/Typography";
-import ThumbUpIcon from "@mui/icons-material/ThumbUp";
-import ThumbDownIcon from "@mui/icons-material/ThumbDown";
-import IconButton from "@mui/material/IconButton";
+import { useState } from "react";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import { deleteComment } from "../../api";
+import { MenuItem, Menu, Button, Fab, Card } from "@mui/material";
+
+function CommentCard({ author, body, date, votes, comment_id }) {
+  const handleDelete = (event) => {
+    deleteComment(comment_id).then(() => {
+      location.reload();
```

Comment:
This is not how we should be updated the comments. If a user is on a slower connection this could result in a long re-load. Ideally this is the point where the comments state is updated to remove the comment that was selected instead.